enclave-runtime/src/attestation.rs

```diff
@@ -499,9 +497,6 @@ pub fn create_ra_report_and_signature<A: EnclaveAttestationOCallApi>(
 
 #[no_mangle]
 pub unsafe extern "C" fn perform_ra(
-	genesis_hash: *const u8,
-	genesis_hash_size: u32,
-	nonce: *const u32,
```

Comment:
This file was completely refactored in PR #971. So I would wait for now, and do the work again, once #971 is merged into master (or rebase this PR onto that branch)